codecov.yml

```diff
@@ -0,0 +1,9 @@
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+
+  status:
+    project: no
+    patch: yes
+    changes: no
```

Comment:
I'd love it if there was any way to avoid adding this config file (I wonder if we could write it only during the travis script itself????) or else setup the config in the "coverage" service directly)  But if there's no way around then I'm okay with us including it.